nilearn/maskers/nifti_labels_masker.py

```diff
@@ -570,25 +563,6 @@ def fit(self, imgs=None, y=None):
 
         return self
 
-    def _init_report_content(self):
-        """Initialize report content.
-
-        Prepare basing content to inject in the HTML template
-        during report generation.
-        """
-        if not hasattr(self, "_report_content"):
-            self._report_content = {
-                "description": (
-                    "This reports shows the regions "
-                    "defined by the labels of the mask."
-                ),
-                "warning_message": None,
-                "number_of_regions": 0,
-            }
-
-        if not hasattr(self, "_reporting_data"):
-            self._reporting_data = None
-
```

Comment:
Unless the model is fitted, the estimator won't have `_reporting_data` and this can be checked using `has_report_data` function provided with `ReportingMixin`.